tests/ballerina-spec-conformance-tests/S05_behavioral_values/tests/objects/client_object_declaration.bal

```diff
@@ -126,7 +126,7 @@ private remote function ClientObject.privateRemoteMethodDecl(string argOne, int
 }
 
 remote function ClientObject.defaultVisibiltyRemoteMethodDecl(string argOne, int argTwo) {
```

Comment:
Should we rename these functions from `XxxDecl` to `XxxDefn` ?

Reply:
This is an outside implementation of a method declaration. Hence the name XxxDecl

Reply:
Got it.